### DISCUSSION TITLE
Mixed line endings in ElectricDevices.csv and GunPoint.csv causing autofix by global gitattributes

### DISCUSSION THREAD

**Discussion by yarnabrina** [alan-turing-institute/sktime#4446] [General]
These are the line endings of all CSV files in `sktime` repository as of 4be2622:

```console
❯ git ls-files --eol | grep -E "*.csv"
i/lf    w/lf    attr/                   sktime/datasets/data/Airline/Airline.cs
i/lf    w/lf    attr/                   sktime/datasets/data/Longley/Longley.cs
i/lf    w/lf    attr/                   sktime/datasets/data/Lynx/Lynx.csv
i/lf    w/lf    attr/                   sktime/datasets/data/PBS_dataset/PBS_dataset.csv
i/lf    w/lf    attr/                   sktime/datasets/data/ShampooSales/ShampooSales.csv
i/lf    w/lf    attr/                   sktime/datasets/data/Uschange/Uschange.csv
i/mixed w/mixed attr/                   sktime/datasets/data/segmentation/ElectricDevices.csv
i/mixed w/mixed attr/                   sktime/datasets/data/segmentation/GunPoint.csv
i/lf    w/lf    attr/                   sktime/datasets/data/solar/solar.csv
```

All but two aforementioned CSV files have LF line endings, and it's causing to auto fix these if someone has global gitattributes to handle such things. For example, I use [this](https://raw.githubusercontent.com/alexkaratarakis/gitattributes/master/Common.gitattributes). This constantly shows me two modified files, and stash/rebase everything gets complicated just a little bit.

If anyone else has faced such problems before, can you please suggest a way to skip global configuration for a specific repository?

(A possible alternative will be to make all LF in `sktime`, but it doesn't seem to be a issue with others so probably not worth the change.)

**Comment by fkiraly** [comment]
I suppose it *would* make sense to change the two files for consistency?

I don't have this problem, but I don't think it will make anyone's life *worse* if we make the endings uniform:

* if none of the types cause problems, that won't change
* if one of the types causes problems, it will be fixed or won't changes
* if both types cause problems, that won't thange.

**Comment by yarnabrina** [comment]
Closed by https://github.com/sktime/sktime/pull/4452
